components/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elasticpath/store-components",
-  "version": "1.1.9",
+  "version": "1.1.10",
```

Comment:
Need to increment this version number

Reply:
> @aChanEP No further comments! We can merge this once the config is updates, then we'll continue with documentation this sprint. Thanks for contributing this one!

Sounds good, I'll place empty values there.

Reply:
Empty values in the config have been pushed